i18n/es/docusaurus-plugin-content-docs/current/interview/interviewer/about.mdx

```diff
@@ -46,14 +46,14 @@ una tabla con consejos generales a la hora de tomar el papel de entrevistador ba
 | ✅ | Hablar claramente y preguntar si entendió cualquier pregunta o problema descrito. |
 | ✅ | Animar al entrevistado a comunicar todo lo que piense o tenga duda. |
 | ⚠️ | Dar pistas (permitir que el entrevistado piense y se rete, no soltarlas rápidamente). |
-| ⚠️ | Permitir que el entrevistado esté callado (si se pide tiempo para ordenar ideas está bien, pero el final de la entrevista es conocer su proceso de pensamiento, por lo que si pide tiempo para pensar no está cumpliendo el objetivo principal). |
-| ✅ | Escribir retroalimentación o marcas de tiempo relevantes para decirlo al final. |
+| ⚠️ | Permitir que el entrevistado esté callado (si se pide tiempo para ordenar ideas está bien, pero el objetivo de la entrevista es conocer su proceso de pensamiento, por lo que si pide tiempo para pensar no está cumpliendo el objetivo principal). |
```

Comment:
en este texto no se termina de entender si el hecho de que el entrevistado pida tiempo es bueno o es malo. Sugiero que clarifiquemos en la parte final con: `... por lo que se deben evitar periodos largos de silencio, de no ser asi, no se podría cumplir con el objetivo`

Reply:
@akotadi

Reply:
> en este texto no se termina de entender si el hecho de que el entrevistado pida tiempo es bueno o es malo. Sugiero que clarifiquemos en la parte final con: `... por lo que se deben evitar periodos largos de silencio, de no ser asi, no se podría cumplir con el objetivo`

@searleser97 @abiisnn este cambio me parece más relevante, creo que cubre lo que habíamos mencionado del tiempo al mencionar `evitar largos periodos de tiempo` aunque sigue siendo un poco ambiguo.

Reply:
Fixed in latest commit